### DISCUSSION TITLE
npm registry is returning 404 for packages that should exist

### DISCUSSION THREAD

**Discussion by candrews** [npm/feedback#960] [Registry] [answered]
Right now, some packages are not installable from npm. There seems to be a bunch of packages missing from registry.npmjs.org that do appear on the npm web site. Some I've noticed include [`@vitest/coverage-v8`](https://www.npmjs.com/package/@vitest/coverage-v8) and [`@microsoft/sarif-multitool`](https://www.npmjs.com/package/@microsoft/sarif-multitool)

To reproduce this problem:
1. `docker run -it --entrypoint /bin/bash node`
2. `npm install -g @microsoft/sarif-multitool`

Actual:
```
npm notice 
npm notice New minor version of npm available! 9.7.2 -> 9.8.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v9.8.0
npm notice Run npm install -g npm@9.8.0 to update!
npm notice 
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@microsoft%2fsarif-multitool - Not found
npm ERR! 404 
npm ERR! 404  '@microsoft/sarif-multitool@*' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in: /root/.npm/_logs/2023-07-14T20_31_43_072Z-debug-0.log
```
Note that https://registry.npmjs.org/@microsoft%2fsarif-multitool displays a 404 error, here's the response body:
```json
{

      "error": "Not found"

}
```

Expected:

The package should install successfully, as it did yesterday.

**Comment by MylesBorins** [comment, marked as answer]
We had an [incident on Friday](https://status.npmjs.org/incidents/jglnklhspsnm). Everything is working now 👍🏼
